lib/graphiti/types.rb

```diff
@@ -101,6 +101,13 @@ def self.map
             kind: "scalar",
             description: "Base Type. Like a normal string, but by default only eq/!eq and case-sensitive.",
           },
+          enum: {
```

Comment:
What about integer enum?

Reply:
are you saying add a second type for that?

Reply:
I think either this is a baseline enum that can work for both strings and integers (considering you're not doing case sensitivity anyway), or it should be a `string_enum` type.